### PR TITLE
DEV-2051: add reactive-papi package files

### PR DESCRIPTION
**Pull request by fanzhao1986**
Add reactive-papi package setup files.

DEV-2051

### CODE REVIEW

**Comment by fanzhao1986** [comment]
needs a second check on thoese files, and local tests works fine, if Generated files are stored in python fold, run the python deployment command `python setup sdist` works fine in the project. Needs a discuss on version, and also the python package deployment on pypi server.

File: python/setup.cfg [hunk 1]
```diff
@@ -0,0 +1,43 @@
+[metadata]
+name = reactive-papi
+description = reactive-platform: Reactive Platform flatbuffer api
+long_description = file: README.md
+url = https://www.reactiveexchange.com
```
Comment (markaylett) [inline python/setup.cfg:5]: Should be https://www.reactivemarkets.com

File: python/setup.cfg [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+project_urls =
+    Source=https://github.com/reactivemarkets/platform-api
+    Tracker=https://github.com/reactivemarkets/platform-api/issues
+author = Reactive Platform
+maintainer = Reactivemarkets
+author-email = development@reactiveexchange.com
```
Comment (markaylett) [inline python/setup.cfg:11]: developers@reactivemarkets.com

File: python/setup.cfg [hunk 3]
```diff
@@ -0,0 +1,43 @@
+[metadata]
+name = reactive-papi
+description = reactive-platform: Reactive Platform flatbuffer api
+long_description = file: README.md
+url = https://www.reactiveexchange.com
+project_urls =
+    Source=https://github.com/reactivemarkets/platform-api
+    Tracker=https://github.com/reactivemarkets/platform-api/issues
+author = Reactive Platform
+maintainer = Reactivemarkets
```
Comment (markaylett) [inline python/setup.cfg:10]: "Reactive Markets" for author and maintainer?

Comment (fanzhao1986) [reply python/setup.cfg]: The author and maintainer like `reactivemarkets/developers` which is the name from github.

**Pull request merged**